File: 0001_twoSum.py
class Solution:
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        n = len(nums)
        
        for i in range(0, n, 1):
            a = nums[i]
            for j in range(i+1, n, 1):
                b = nums[j]
                if a+b == target:
                    result = [i, j]
                    return result
                else:
                    continue
        

File: 0001_twoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        int[] output = new int[2];
        for(int i=0; i<n; i++){
            int a=nums[i];
            for(int j=i+1; j<n; j++){
                int b=nums[j];
                if(a+b==target){
                    output[0]=i;
                    output[1]=j;
                    
                }
                else{
                    continue;
                }
            }
        }
        return output;
    }
}
